### ISSUE TITLE
Release 0.2.0 seems to break property prediction

### ISSUE THREAD

**Issue by johnberroa** [DeepGraphLearning/torchdrug#132]
When running the tutorial code found here (https://torchdrug.ai/docs/tutorials/property_prediction.html), I get the following error once I start training:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Input In [5], in <cell line: 3>()
      1 optimizer = torch.optim.Adam(task.parameters(), lr=1e-3)
      2 solver = core.Engine(task, train_set, valid_set, test_set, optimizer, batch_size=1024)
----> 3 solver.train(num_epoch=100)
      4 solver.evaluate("valid")

File /anaconda/envs/azureml_py38/lib/python3.8/site-packages/torchdrug/core/engine.py:155, in Engine.train(self, num_epoch, batch_per_epoch)
    152 if self.device.type == "cuda":
    153     batch = utils.cuda(batch, device=self.device)
--> 155 loss, metric = model(batch)
    156 if not loss.requires_grad:
    157     raise RuntimeError("Loss doesn't require grad. Did you define any loss in the task?")

File /anaconda/envs/azureml_py38/lib/python3.8/site-packages/torch/nn/modules/module.py:1102, in Module._call_impl(self, *input, **kwargs)
   1098 # If we don't have any hooks, we want to skip the rest of the logic in
   1099 # this function, and just call forward.
   1100 if not (self._backward_hooks or self._forward_hooks or self._forward_pre_hooks or _global_backward_hooks
   1101         or _global_forward_hooks or _global_forward_pre_hooks):
-> 1102     return forward_call(*input, **kwargs)
   1103 # Do not call functions when jit is used
   1104 full_backward_hooks, non_full_backward_hooks = [], []

File /anaconda/envs/azureml_py38/lib/python3.8/site-packages/torchdrug/tasks/property_prediction.py:107, in PropertyPrediction.forward(self, batch)
    105     loss = F.mse_loss(pred, target, reduction="none")
    106 elif criterion == "bce":
--> 107     loss = F.binary_cross_entropy_with_logits(pred, target, reduction="none")
    108 elif criterion == "ce":
    109     loss = F.cross_entropy(pred, target.long().squeeze(-1), reduction="none").unsqueeze(-1)

File /anaconda/envs/azureml_py38/lib/python3.8/site-packages/torch/nn/functional.py:2980, in binary_cross_entropy_with_logits(input, target, weight, size_average, reduce, reduction, pos_weight)
   2977     reduction_enum = _Reduction.get_enum(reduction)
   2979 if not (target.size() == input.size()):
-> 2980     raise ValueError("Target size ({}) must be the same as input size ({})".format(target.size(), input.size()))
   2982 return torch.binary_cross_entropy_with_logits(input, target, weight, pos_weight, reduction_enum)

ValueError: Target size (torch.Size([1024, 2])) must be the same as input size (torch.Size([1024, 4]))
```

Downgrading to 0.1.3 properly trains and throws no error.

I came across this error first when trying to train on my own dataset, with a boolean target column. The error I got there was similar:

```
ValueError: Target size (torch.Size([32, 1])) must be the same as input size (torch.Size([32, 2]))
```

It seems like the input size is * 2 the target size.

Has the general property prediction workflow changed with the new release?

**Comment by KiddoZhu** [comment]
Hi! The property prediction part has been changed a little bit to include some more arguments and datasets with many classification targets, but generally we want to make it compatible with old code and tutorials. I will take a look at that.

@Oxer11 Do you have any idea what the problem is here?

**Comment by KiddoZhu** [comment]
Fixed in 597f43b
